package/MDAnalysis/analysis/gnm.py

```diff
@@ -245,8 +245,18 @@ class GNMAnalysis(AnalysisBase):
        Use :class:`~MDAnalysis.analysis.AnalysisBase` as parent class and
        store results as attributes ``times``, ``eigenvalues`` and
        ``eigenvectors`` of the ``results`` attribute.
+
+    .. versionchanged:: 2.8.0
+       introduced a :meth:`get_supported_backends` allowing for execution on with
+       ``multiprocessing`` and ``dask`` backends.
```

Comment:
fix grammar and make it more obvious to users what happened (I know, this is a copy of the RMSD one... should have caught it then — perhaps fix it, too in this PR?)
```suggestion
    .. versionchanged:: 2.8.0
       Enabled **parallel execution** with the ``multiprocessing`` and ``dask`` 
       backends; use the new method :meth:`get_supported_backends` to see all 
       supported backends.
```